quickstart.ps1

```
@@ -1112,6 +1114,12 @@ try {
     if ($LASTEXITCODE -eq 0) { $OllamaDetected = $true }
 } catch { }
 
+$LlamaCppDetected = $false
+try {
+    $response = Invoke-RestMethod -Uri "http://localhost:8080/v1/models" -ErrorAction SilentlyContinue
+    if ($response) { $LlamaCppDetected = $true }
+} catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)